data/datavalid.yml

```diff
@@ -452,5 +400,46 @@ schemas:
       - name: stop_and_search_time
       - name: agency
       - name: uid
+  uof_citizen:
+    columns:
+      - name: uof_citizen_uid
+        description: >
+          unique identifier of citizen.
+      - name: uof_uid
+        description: >
+          unique iidentifer of use of force report.
+      - name: citizen_influencing_factors
+      - name: citizen_distance_from_officer
+      - name: citizen_arrested
+      - name: citizen_arrest_charges
+      - name: citizen_hospitalized
+      - name: citizen_injured
+      - name: citizen_build
+      - name: citizen_height
+      - name: citizen_sex
+      - name: agency
+  uof_officer:
+    columns:
+      - name: uid
+        description: >
+          unique identifier of officer.
+      - name: uof_uid
+        description: >
+          unique identifier of use of force report.
+      - name: first_name
+      - name: last_name
+      - name: use_of_force_type
+      - name: use_of_force_level
+      - name: use_of_force_effective
+      - name: use_of_force_reason
+      - name: race
+      - name: sex
```

Comment:
`first_name`, `last_name`, `race`, and `sex` never change so maybe we can just put them in the `personnel` table.

Reply:
Thanks.